phoenix-core/src/test/java/org/apache/phoenix/parse/QueryParserTest.java

```diff
@@ -289,24 +289,6 @@ public void testNegativeCountStar() throws Exception {
     }
 
     @Test
-    public void testUnknownFunction() throws Exception {
```

Comment:
Now the exception will be thrown during compilation not while parsing because while parsing not able check whether udf enabled or not. That's why removed it.